worker.go

```diff
@@ -49,12 +50,18 @@ type (
 		// default: defaultMaxConcurrentActivityExecutionSize(1k)
 		MaxConcurrentActivityExecutionSize int
 
-		// Optional: Sets the rate limiting on number of activities that can be executed per second.
+		// Optional: Sets the rate limiting on number of activities that can be executed per refresh duration.
 		// This can be used to protect down stream services from flooding.
 		// The zero value of this uses the default value.
 		// default: defaultMaxActivityExecutionRate(100k)
 		// Warning: activity's StartToCloseTimeout starts ticking even if a task is blocked due to rate limiting.
-		MaxActivityExecutionsPerSecond float32
```

Comment:
we don't use float (fraction), it is not per 'second' anymore. It is per refresh duration. You can set rate to 1 and duration to 10s which means 0.1 RPS.